logstash-output-lmlogs.gemspec

```diff
@@ -1,6 +1,6 @@
 Gem::Specification.new do |s|
   s.name = 'logstash-output-lmlogs'
-  s.version         = '1.0.2'
+  s.version         = '1.0.3'
```

Comment:
Should this be 1.1.0 as this is not a bugfix ?

Reply:
Yes. Done